control/control.rnc

```diff
@@ -360,6 +361,13 @@ software_packages = element packages {
     element package { text }+
 }
 
+# the default preselected products (modules) for the base product, used only
+# in Full medium offline installation, see https://jira.suse.com/browse/SLE-8040
+software_default_modules = element default_modules {
+    LIST,
+    element default_module { text }*
```

Comment:
why `*` and not `+` ?

Reply:
In theory you might need to override the default `control.xml` value (`["sle-module-basesystem"]`) with an empty list.